assembly/env/env.ts

```diff
@@ -43,10 +43,13 @@ export namespace env {
   export declare function createSC(bytecode: StaticArray<u8>): string;
 
   @external("massa", "assembly_script_get_keys")
-  export declare function getKeys(): StaticArray<u8>;
+  export declare function getKeys(prefix?: string): StaticArray<u8>;
```

Comment:
Thanks, I will add a default value. I choose string because address in assembly_script_get_keys_for is also a string and in my opinion it is weird on our side to have the address as a string and the prefix in StaticArray<u8>.

Reply:
ok just saw sc-runtime. I'll put StaticArray